### ISSUE TITLE
uniDisplay descriptors incomplete

### ISSUE THREAD

**Issue by blakadder** [tasmota/docs#1030]
@gemu2015 

I've been trying to create a descriptor for a seemingly simple SSD1306 0.42" screen but the documentation was leading in me in the wrong direction.

Can you explain in more detail the options under `:A` and why it says its optional and color display only since I **had** to use it for the above mentioned screen?

It also seems the example for SH1106 has an error

**Comment by gemu2015** [comment]
indeed this documentation is old and sometimes outdated.
:A is needed for most displays not only color.

i will update the docs soon.

however the SH1106 and SD1306 descriptors from the tasmota descriptor folder work for me without problems.

**Comment by blakadder** [comment]
shouldn't this https://github.com/arendst/Tasmota/blob/6400c9d650d385f5082f9d3557314c9da9ca77a9/tasmota/displaydesc/SD1306_display.ini#L15 have a `,`

**Comment by gemu2015** [comment]
yes you are right, strange that my display worked. however i tested this a long time before. i don't use SD1306 any more
the SH1106 descriptor is in use and works, i just downloaded the descriptor to a device with success

**Comment by blakadder** [comment]
no problem, i tested the SSD1306 yesterday, i'll make the needed changes... that part of the code enables `pre-charge period 0x022/f1*/` whatever that is and the screen does work without it
